### ISSUE TITLE
Ability to install and run modules

### ISSUE THREAD

**Issue by FFY00** [pipxproject/pipx#517]
**How would this feature be useful?**
This would be useful for projects that don't have an entrypoint but have a runnable module.

**Describe the solution you'd like**
Add a `-m`/`--module` argument to `pipx install` to install as a module and a `-m` argument to `pipx` to run the module.

Example
```
$ pipx install -m build  # install build as module
$ pipx -m build  # run the build module
```

The pip/Python equivalent
```
$ pip install build
$ python -m build
```

Related: https://github.com/FFY00/python-build/issues/101

**Comment by layday** [comment]
Probably a duplicate of https://github.com/pipxproject/pipx/issues/177, which was filed for build's precursor.

**Comment by cs01** [comment]
@gaborbernat please add a comment explaining why you close an issue before you do so

**Comment by gaborbernat** [comment]
Merged the topic in #177, which is the original request.
